site/docs/v1/tech/shared/_recaptcha-values.adoc

```diff
@@ -0,0 +1,4 @@
+ * `GoogleRecaptchaV2` - use Google Recaptcha v2
+ * `GoogleRecaptchaV3` - use Google Recaptcha v3
+ * `HCaptcha` - use HCaptcha
+ * `HCaptchaEnterprise` - use HCaptcha Enterprise - v25
```

Comment:
Can we fix up any reference of `ReCaptcha` to be the correct Google name of `reCAPTCHA` ? The enum values are fine, but any text version should use `reCAPTCHA`.